packages/ketcher-react/src/script/ui/state/hotkeys.ts

```diff
@@ -123,9 +123,18 @@ function getHoveredItem(
   const hoveredItem = {}
 
   for (const ctabItem in ctab) {
-    if (!(ctab[ctabItem] instanceof Map)) continue
+    if (Object.keys(hoveredItem).length) {
+      break
+    }
+
+    if (!(ctab[ctabItem] instanceof Map)) {
+      continue
+    }
+
     ctab[ctabItem].forEach((item, id) => {
-      if (item.hover) hoveredItem[ctabItem] = id
+      if (item.hover) {
+        hoveredItem[ctabItem] = id
```

Comment:
`return hoveredItem`

And then we don't need 1st if (126 str) and don't need ternary operator below

Reply:
You cannot `return` from a forEach loop.
And you cannot use another iterator for a Map.
I can turn it into in Array, go with `for_in` and return from there, but it will be much less readable than it is now.